src/main/java/org/testeditor/fixture/web/WebDriverFixture.aml

```diff
@@ -7,19 +7,19 @@ interaction type wait {
 
 interaction type startBrowser {
 	label = "Start browser"
-	template = "start browser" ${element} 
+	template = "Start browser" ${element} 
```

Comment:
Shouldn't this be "Start" ${element}, since template for firefox portable is not "Start browser firefox protable" ${path}